File: src/main/java/io/beaniejoy/dongnecafe/domain/cafe/dto/cafe/CafeImageResponseDto.java
package io.beaniejoy.dongnecafe.domain.cafe.dto.cafe;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class CafeImageResponseDto {

    private String imgUrl;
}

File: src/main/java/io/beaniejoy/dongnecafe/domain/cafe/dto/cafe/CafeImageResponseDto.kt
package io.beaniejoy.dongnecafe.domain.cafe.dto.cafe

import io.beaniejoy.dongnecafe.domain.cafe.domain.CafeImage

data class CafeImageResponseDto(
    val imgUrl: String? = null
) {
    companion object {
        fun of(cafeImage: CafeImage): CafeImageResponseDto {
            return CafeImageResponseDto(cafeImage.imgUrl)
        }
    }
}